### PR TITLE
Animations: Enable Animations.

### PR DESCRIPTION
**Pull request by carloskelly13**
## Summary

Removes the feature flag for animations.

## Testing Instructions

1. Create a new story in the workspace and add a bunch of shapes and pictures
2. In the animation panel on the right, add each kind of animation to verify each works.
3. For some animations there are various directions, the animation could perform the direction of the picker
4. Each animation should have a duration and delay and should execute based on those timings

---

#5556

### CODE REVIEW

File: assets/src/edit-story/output/page.js [hunk 1]
```diff
@@ -38,12 +38,10 @@ import getLongestMediaElement from './utils/getLongestMediaElement';
 const ASPECT_RATIO = `${PAGE_WIDTH}:${PAGE_HEIGHT}`;
 
 function OutputPage({ page, autoAdvance, defaultPageDuration }) {
-  const enableAnimation = useFeature('enableAnimation');
```
Comment (spacedmonkey) [inline assets/src/edit-story/output/page.js:41]: The linting error seems valid here.

File: .storybook/preview.js [hunk 2]
```diff
@@ -90,25 +90,23 @@ addDecorator((story, { id }) => {
...
   if (isDashboardStorybook) {
     return (
-      <FlagsProvider features={{ enableAnimation: true }}>
-        <ThemeProvider
-          theme={{
-            DEPRECATED_THEME: dashboardTheme,
-            ...designSystemTheme,
-            colors: lightMode,
-          }}
+      <ThemeProvider
```
Comment (spacedmonkey) [inline .storybook/preview.js:92]: Linter error also valid here.

**Review by BrittanyIRL — APPROVED** [review]
This works as expected! no animation flag necessary any longer to see animations in the editor. 🚀

**Review by spacedmonkey — APPROVED** [review]
👍

**Comment by spacedmonkey** [comment]
Shouldn't there be details on how to QA animations.

**Comment by carlos-kelly** [comment]
@spacedmonkey For this one and pre-publish, we are asking QA to review the feature briefs to QA the feature as a whole piece (to ensure it works together). It's in the parent ticket.

**Review by maxyinger — APPROVED** [review]

**Pull request merged**